### PR TITLE
Add long description to package information

### LINKED ISSUE #567: Add description to PyPI package
Currently there is no description for the package on PyPI: https://pypi.org/project/raiden-services/#description

We should add that.

### PR DESCRIPTION
**Pull request by palango**
Fixes #567

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -29,15 +28,16 @@ def read_requirements(path: str) -> List[str]:
...
+with open("README.md") as readme_file:
+    README = readme_file.read()
 
 
 setup(
     name="raiden-services",
     version="0.4.0",
     license="MIT",
     description=DESCRIPTION,
+    long_description=README,
```
Comment (karlb) [inline setup.py]: Do we need to add
```
long_description_content_type='text/markdown'
```
?

Comment (palango) [reply setup.py]: Good point, I missed that in the raiden setup config.

**Comment by karlb** [comment]
It looks like this is causing a docker build failure: https://cloud.docker.com/u/raidennetwork/repository/registry-1.docker.io/raidennetwork/raiden-services/builds/f03de662-9c3b-44aa-ae75-c811d3696036

**Comment by palango** [comment]
Mh, this error message doesn't tell me much. Any ideas?

```
 ERROR: Command errored out with exit status 1:
command: /usr/local/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/services/setup.py'"'"'; __file__='"'"'/services/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info
```

**Comment by karlb** [comment]
The interesting part is
```
Complete output (5 lines):
Traceback (most recent call last):
File "<string>", line 1, in <module>
File "/services/setup.py", line 31, in <module>
with open("README.md") as readme_file:
FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
----------------------------------------
```

**Pull request merged**